packages/vscode-extension/src/plugins/network/strategies/NewInspectorStrategy.ts

```diff
@@ -0,0 +1,171 @@
+import { commands, Disposable } from "vscode";
+import { NetworkPlugin } from "../network-plugin";
+import { disposeAll } from "../../../utilities/disposables";
+import { Logger } from "../../../Logger";
+import {
+  CDPMessage,
+  NetworkMethod,
+  WebviewCommand,
+  WebviewMessage,
+} from "../../../network/types/panelMessageProtocol";
+import { BaseInspectorStrategy } from "./BaseInspectorStrategy";
+
+import { NETWORK_EVENTS } from "../../../network/types/panelMessageProtocol";
+import { NETWORK_EVENT_MAP, NetworkBridge, RadonInspectorBridge } from "../../../project/bridge";
+import { ResponseBodyData } from "../../../network/types/network";
+
+// Truncation constants
+const MAX_MESSAGE_LENGTH = 1000000;
+const TRUNCATED_LENGTH = 1000000;
+
+export default class NewInspectorStrategy extends BaseInspectorStrategy {
```

Comment:
`New` and `Legacy` aren't very helpful for a person which doesn't have all the context of how the implementations came to be.
Maybe `DebuggerNetworkInspector` and `InspectorBridgeNetworkInspector` (based on the underlying communication channel)?